### PR TITLE
[ADD] website: mail groups

### PR DESCRIPTION
**Pull request by auva-odoo**
task-2845181

### CODE REVIEW

**Review by toaa-odoo — APPROVED** [review]
Hey @auva-odoo, great work! I only moved a line, the rest was all good :)

File: content/applications/websites/website/mail_groups.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Configuring mail groups
+=======================
+
+To configure mail groups, proceed as follows:
+
+#. Configure a custom email alias domain: Access the **General settings**, scroll down to the
+   :guilabel:`Discuss` section, enable the :guilabel:`Custom Email Server` feature, and enter the
+   :guilabel:`Alias domain` (e.g., `@mycompany.com`).
+#. Go to :menuselection:`Website --> Groups --> Mailing list groups`, then click :guilabel:`Create`.
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:58]:
```suggestion
#. Go to :menuselection:`Website --> Groups --> Mailing List Groups`, then click :guilabel:`Create`.
```

File: content/applications/websites/website/mail_groups.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+#. Specify a :guilabel:`Group name`, the :guilabel:`Email alias`, and a :guilabel:`Description`.
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:59]:
```suggestion
#. Specify a :guilabel:`Group Name`, the :guilabel:`Email Alias`, and a :guilabel:`Description`.
```

File: content/applications/websites/website/mail_groups.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+#. In the :guilabel:`Privacy` tab, define who can subscribe to the mail group:
+
+   - :guilabel:`Everyone`: to make the mail group public so anyone can subscribe to it;
+   - :guilabel:`Members only`: to only allow users defined as members to subscribe to the mail group;
+   - :guilabel:`Selected group of users`: to only allow users from the :guilabel:`Authorized group`
+     to subscribe to the mail group.
+
+#. If the mail group is moderated, you can automatically notify authors when their message is
+   pending moderation by enabling :guilabel:`Automatic notification` in the :guilabel:`Notify
+   members` tab and writing the :guilabel:`Notification message`.
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:73]:
```suggestion
   Members` tab and writing the :guilabel:`Notification message`.
```

File: content/applications/websites/website/mail_groups.rst [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+#. If you wish to send out guidelines to new subscribers, enable :guilabel:`Send guidelines to new
+   subscribers` and write the :guilabel:`Guidelines` in the :guilabel:`Guidelines` tab. This is
+   particularly useful when the mail group is moderated.
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:76]:
```suggestion
#. If you wish to send out guidelines to new subscribers, enable :guilabel:`Send guidelines to new
   subscribers` and write them in the :guilabel:`Guidelines` tab. This is
   particularly useful when the mail group is moderated.
```

File: content/applications/websites/website/mail_groups.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+.. _website/mailing_lists/moderate:
+
+Moderating mail group messages
+==============================
+
+If the :guilabel:`Moderate this group` feature has been enabled for the
+:ref:`mail group <website/mailing_lists/configure_groups>`, one of the :guilabel:`Moderators` must
+approve the group's messages before they are dispatched to the other members.
+
+To moderate messages, go to :menuselection:`Website --> Groups --> Mailing list groups`, select the
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:87]:
```suggestion
To moderate messages, go to :menuselection:`Website --> Groups --> Mailing List Groups`, select the
```

File: content/applications/websites/website/mail_groups.rst [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+mail group, and click the :guilabel:`To review` smart button. You can moderate messages using the
+buttons at the end of the message line or select a message to view its content and moderate it
+accordingly.
+
+   .. image:: mail_groups/mail-group-moderation.png
+      :alt: Moderation buttons in the message line.
+
+The following buttons are available:
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:95]:
```suggestion
The following actions are available:
```

File: content/applications/websites/website/mail_groups.rst [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  email address with the status :guilabel:`Always allow`.
+- :guilabel:`Ban`: to blacklist the author, i.e. automatically discard all their emails. In the
+  pop-up window that opens, click :guilabel:`Ban` to ban the author without notifying them, or
+  specify an explanation, then click :guilabel:`Send & Ban` to ban the author and send them the
+  explanation. As a result, a :ref:`moderation rule <website/mailing_lists/moderate>` is created for
+  the author's email address with the status :guilabel:`Permanent ban`.
+
+.. note::
+   Messages can also be moderated from the group's list of messages. Go to :menuselection:`Website
+   --> Groups --> Mailing list groups`, select the mail group and click the :guilabel:`Emails` smart
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:113]:
```suggestion
   --> Groups --> Mailing List Groups`, select the mail group and click the :guilabel:`Emails` smart
```

File: content/applications/websites/website/mail_groups.rst [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+   button.
+
+.. _website/mailing_lists/moderation_rules:
+
+Whitelisting/Blacklisting authors
+=================================
+
+You can whitelist or blacklist an author either directly :ref:`from a mail group message
+<website/mailing_lists/moderate>`, or by creating a moderation rule. To do so, go to
+:menuselection:`Website --> Groups --> List moderation rulings` and click :guilabel:`Create`. Then,
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:123]:
```suggestion
:menuselection:`Website --> Groups --> List Moderation Rulings` and click :guilabel:`Create`. Then,
```

File: content/applications/websites/website/mail_groups.rst [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+select the :guilabel:`Group`, specify the author's :guilabel:`Email` and set the :guilabel:`Status`
+field.
+
+.. tip::
+   You can also access the mail group's moderation rules by going to :menuselection:`Website -->
+   Groups --> Mailing list groups`, selecting the group, then clicking the :guilabel:`Moderations`
```
Comment (xpl-odoo) [inline content/applications/websites/website/mail_groups.rst:129]:
```suggestion
   Groups --> Mailing List Groups`, selecting the group, then clicking the :guilabel:`Moderations`
```

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Great job @auva-odoo ! Almost all comments are nitpicks, as usual ;)

Note: do we really want to have this page at the level of the Pages/Configuration etc. sections? It feels like we give this feature too much importance maybe?

**Comment by auva-odoo** [comment]
Hey @xpl-odoo thank you for the review! I implemented all of the suggested changes, except the one regarding weather we really want to have this page at the level of the Pages/Configuration etc. sections? I would argue that although I see your point, it's the solution that makes the most sense to me at this time. We can discuss this if you want :-)

**Comment by xpl-odoo** [comment]
> Hey @xpl-odoo thank you for the review! I implemented all of the suggested changes, except the one regarding weather we really want to have this page at the level of the Pages/Configuration etc. sections? I would argue that although I see your point, it's the solution that makes the most sense to me at this time. We can discuss this if you want :-)

No need to discuss this, we can @robodoo r+ :)

**Comment by xpl-odoo** [comment]
> @robodoo r+

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**